nanoFirmwareFlasher/nanoFirmwareFlasher.csproj

```diff
@@ -19,19 +19,19 @@
     <Description>.NET nanoFirmwareFlasher tool to flash firmware images to target devices.</Description>
     <!-- need this to allow async Main() -->
     <LangVersion>latest</LangVersion>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>netcoreapp5;net48</TargetFrameworks>
```

Comment:
are you sure about netcoreapp5? why not juste net5?

Reply:
Good question... It's like that since the move to a dotnet tool... not 100% sure if that's not a requirement to be a dotnet tool... do you?